### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by raveling**
## Description

Adding a newly built Gatsby site to the showcase.

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -9859,3 +9859,16 @@
     - Agency
   built_by: App Design
   built_by_url: "https://appdesign.dev/"
+  - title: Guillermo Gómez-Peña
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
- title: Guillermo Gómez-Peña
```

Comment (raveling) [reply docs/sites.yml]: Sorry I missed that, and thanks for the suggestion!

File: docs/sites.yml [hunk 2]
```diff
@@ -9859,3 +9859,16 @@
     - Agency
   built_by: App Design
   built_by_url: "https://appdesign.dev/"
+- title: Guillermo Gómez-Peña
+  url: https://www.guillermogomezpena.com/
+  main_url: https://www.guillermogomezpena.com/
+  source_url:
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
```
Missed this. The null will also cause problems so better to remove it.

**Comment by laurieontech** [comment]
Thanks for the PR! For some reason this submission was having issues when merging so I had to do it manually. The commit can be found here: https://github.com/gatsbyjs/gatsby/commit/a30bcf6d9f8ea23b80b137bb3bd1555dccbca845

Appreciate your submission.

**Pull request merged**